scripts/jenkins/jobs-obs/openstack-repocheck.yaml

```diff
@@ -37,19 +37,20 @@
             - SLE_12_SP2
             - openSUSE_13.2
             - openSUSE_Leap_42.1
+            - openSUSE_Leap_42.2
       - axis:
           type: slave
           name: nodes
           values:
             - cloud-trackupstream
     execution-strategy:
       combination-filter: |
-       !(["Cloud:OpenStack:Master"].contains(project) && ["SLE_11_SP3", "SLE_12", "openSUSE_13.2"].contains(repository) ||
-         ["Cloud:OpenStack:Factory"].contains(project) && ["SLE_11_SP3", "SLE_12", "SLE_12_SP1", "SLE_12_SP2", "openSUSE_Leap_42.1"].contains(repository) ||
-         ["Cloud:OpenStack:Juno", "Cloud:OpenStack:Juno:Staging"].contains(project) && ["SLE_12_SP1", "SLE_12_SP2", "openSUSE_Leap_42.1"].contains(repository)  ||
-         ["Cloud:OpenStack:Kilo", "Cloud:OpenStack:Kilo:Staging"].contains(project) && ["SLE_12_SP1", "SLE_12_SP2", "openSUSE_Leap_42.1", "SLE_11_SP3"].contains(repository)  ||
-         ["Cloud:OpenStack:Liberty", "Cloud:OpenStack:Liberty:Staging"].contains(project) && ["SLE_11_SP3", "SLE_12_SP2"].contains(repository) ||
-         ["Cloud:OpenStack:Mitaka", "Cloud:OpenStack:Mitaka:Staging"].contains(project) && ["openSUSE_13.2", "SLE_12", "SLE_11_SP3"].contains(repository) ||
+       !(["Cloud:OpenStack:Master"].contains(project) && ["SLE_11_SP3", "SLE_12", "openSUSE_13.2", "openSUSE_Leap_42.2"].contains(repository) ||
+         ["Cloud:OpenStack:Factory"].contains(project) && ["SLE_11_SP3", "SLE_12", "SLE_12_SP1", "SLE_12_SP2", "openSUSE_Leap_42.1", "openSUSE_Leap_42.2"].contains(repository) ||
+         ["Cloud:OpenStack:Juno", "Cloud:OpenStack:Juno:Staging"].contains(project) && ["SLE_12_SP1", "SLE_12_SP2", "openSUSE_Leap_42.1", "openSUSE_Leap_42.2"].contains(repository)  ||
+         ["Cloud:OpenStack:Kilo", "Cloud:OpenStack:Kilo:Staging"].contains(project) && ["SLE_12_SP1", "SLE_12_SP2", "openSUSE_Leap_42.1", "SLE_11_SP3", "openSUSE_Leap_42.2"].contains(repository)  ||
+         ["Cloud:OpenStack:Liberty", "Cloud:OpenStack:Liberty:Staging"].contains(project) && ["SLE_11_SP3", "SLE_12_SP2", "openSUSE_Leap_42.2"].contains(repository) ||
+         ["Cloud:OpenStack:Mitaka", "Cloud:OpenStack:Mitaka:Staging"].contains(project) && ["openSUSE_13.2", "SLE_12", "SLE_11_SP3", "openSUSE_Leap_42.2"].contains(repository) ||
```

Comment:
because noone created it so far, asked for it and fixed it. given the sorry state of 42.1 in Mitaka and given that Mitaka is an interim release that we do not maintain for a longer amount of time I wouldn't want to spend time on it.